packages/cardano-services/src/Http/HttpServer.ts

```diff
@@ -93,6 +93,15 @@ export class HttpServer extends RunnableModule {
       );
     }
 
+    const requestLogger = contextLogger(this.logger, 'request');
+
+    this.app.use((req, _res, next) => {
+      const { body, method, path, query } = req;
+
+      requestLogger.debug({ body, method, path, query, time: Date.now() });
```

Comment:
I am not sure if the `time` part is needed. I think it should/is added by the logger implementation (by bunyan for example).

Reply:
Yes... loggers _usually_ add time by themselves...
Removed in 60c13c5fa